mangaki/mangaki/tests/test_anidb.py

```diff
@@ -47,5 +52,19 @@ def test_anidb_get(self):
             content_type='application/xml'
         )
         anime = self.anidb.get_or_update_work(11606)
-        self.assertNotEqual(anime.title, '')
-        self.assertNotEqual(anime.worktitle_set.count(), 0)
+
+        self.assertEqual(anime.title, 'Sangatsu no Lion')
+        self.assertEqual(anime.nb_episodes, 22)
+        self.assertEqual(anime.studio.title, 'Shaft')
+
+        author_ids = [q.artist_id for q in Staff.objects.filter(work_id=anime.pk, role_id=3)]
+        author_names = [Artist.objects.get(pk=at_id).name for at_id in author_ids]
+        self.assertEqual(author_names, ['Umino Chika'])
```

Comment:
Asserting between lists should be done using [`assertCountEqual`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual) rather than `assertEqual`.

For example, `self.assertEqual(['A', 'B'], ['B', 'A'])` would fail, using `assertCountEqual` on these lists would succeed.